src/server/debug.rs

```diff
@@ -462,6 +484,9 @@ impl Debugger {
             if raft_state.get_last_index() < apply_state.get_applied_index() {
                 return Err(Error::Other("last index < applied index".into()));
             }
+            if raft_state.get_hard_state().get_commit() < apply_state.get_applied_index() {
```

Comment:
Do we need to check `commit_index > last_index`?

Reply:
It's not necessary because they are in 1 rocksdb.

Reply:
If this check fails, it means either rocksdb is corrupted or there is a bug in raft.